core-services/prow/02_config/openshift/cloud-network-config-controller/_pluginconfig.yaml

```diff
@@ -15,7 +15,10 @@ label:
       - dcbw
       - knobunc
       - kyrtapz
-      - trozet
+      - martinkennelly
```

Comment:
why is peri not on this list?

can you please double check all names are correct based on https://docs.google.com/document/d/15LUMFSHayUj54NpVeztDRSEzcQ4zKDpb7b1dyDBKgpE/edit?tab=t.yfhq4z7bkm3u#heading=h.x37k9k86rm3s
I don't have time to check it case by case - another way is to check the OWNERs file in each repo: https://github.com/openshift/cloud-network-config-controller/blob/main/OWNERS

Reply:
> why is peri not on this list?

because I made a mistake and confused another "P" name pliurh w/ Peri. My fault. Thanks for the review.

I've updated this PR to fix it.

> can you please double check all names are correct based on https://docs.google.com/document/d/15LUMFSHayUj54NpVeztDRSEzcQ4zKDpb7b1dyDBKgpE/edit?tab=t.yfhq4z7bkm3u#heading=h.x37k9k86rm3s

yes, this is exactly what I used to create this PR. I just double checked and I think it's right now